src/Elliptic/Systems/ScalarGaussBonnet/FirstOrderSystem.hpp

```diff
@@ -0,0 +1,51 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "Elliptic/BoundaryConditions/BoundaryCondition.hpp"
+#include "Elliptic/Protocols/FirstOrderSystem.hpp"
+#include "Elliptic/Systems/ScalarGaussBonnet/Equations.hpp"
+#include "Elliptic/Systems/Xcts/Tags.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "Utilities/ProtocolHelpers.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace sgb {
+
+/*!
+ * \brief The scalar equation in sGB theories of gravity in the decoupled limit,
+ * assuming quasi-stationarity. See \ref sgb for details on the
+ * explicit equation.
+ */
+struct FirstOrderSystem
+    : tt::ConformsTo<elliptic::protocols::FirstOrderSystem> {
+ private:
+  using field = ::CurvedScalarWave::Tags::Psi;
```

Comment:
Is this tag the only dependence of the whole system on `CurvedScalarWave`? Then you could just define your own `sgb::Tags::Field` and don't worry about using `CurvedScalarWave` things. Then the `CurvedScalarWave` system isn't a dependency anymore and doesn't need to be compiled.